packages/server/src/fhir/operations/expand.test.ts

```diff
@@ -165,18 +165,19 @@ describe.each<Partial<Project>>([{ features: [] }, { features: ['terminology'] }
     });
   });
 
-  test('Handle punctuation', async () => {
-    const res = await request(app)
-      .get(
-        `/fhir/R4/ValueSet/$expand?url=${encodeURIComponent(
-          'http://hl7.org/fhir/ValueSet/observation-codes'
-        )}&filter=${encodeURIComponent('intention - reported')}`
-      )
-      .set('Authorization', 'Bearer ' + accessToken);
-    expect(res.status).toBe(200);
-    expect(res.body.expansion.contains[0].system).toBe(LOINC);
-    expect(res.body.expansion.contains[0].display).toMatch(/pregnancy intention/i);
-  });
+  test.only('Handle punctuation', () =>
```

Comment:
@mattlong Was this change to `test.only` intentional?

Reply:
looks like this was resolved in a followup https://github.com/medplum/medplum/pull/4049/files#diff-a78100eed6392880954d195285f905271825495fe43fc9c8bebe85b6ea5076acR168